docs/topics/loaders.rst

```diff
@@ -225,7 +225,7 @@ Item Loader Context
 
 The Item Loader Context is a dict of arbitrary key/values which is shared among
 all input and output processors in the Item Loader. It can be passed when
-declaring, instantiating or using Item Loader. They are used to modify the
+declaring, instantiating or using Item Loader. They are used to modifying the
```

Comment:
This change sounds odd, I think the tool that was used to make these corrections thought `used to` in this phrase meant "became familiar with". Could somebody verify this?